uint8[13] can_data
